client/jetpack-connect/style.scss

```diff
@@ -937,11 +937,6 @@ $colophon-height: 50px;
 			margin-right: auto;
 		}
 
-		.logged-out-form input:focus {
-			border: 0;
-			box-shadow: none;
-		}
-
```

Comment:
I think it's fine to delete those changes because they are scoped under `is-jetpack-woocommerce-flow` or `is-jetpack-woo-dna-flow`, but I'd ask to the PR reviewers (especially the ones with more knowledge about all possible flows) to thoroughly check in case these styles are still necessary.